Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM oracle/openjdk
```

Comment:
Shouldn't this be referencing a specific tag? to ensure our performance tests always perform the same?

Reply:
Makes sense, I will fix it.

Reply:
Fixed.